site/_config.yml

```diff
@@ -33,6 +33,12 @@ exclude:
   - Gemfile
   - Gemfile.lock
   - vendor
+  - init.go
+  - app.yaml
+  - package.json
+  - service-acc.json.gpg
+  - sw-precache-config.js
+  - node_modules
```

Comment:
It seems like these:
```
app_engine:
  runtime: go
  api_version: go1
  default_expiration: 300s
```
aren't needed any more and can be removed from `_config.yaml`